docs/source/using/addons.adoc

```diff
@@ -76,6 +76,25 @@ If the metadata field *OpenShift-Version* is not specified in the add-on header,
 *OpenShift-Version* only supports versions in the form of <major>.<minor>.<patch>.
 ====
 
+[[addon-defining-addon-dependencies]]
+== Defining Add-on Dependencies
+
+Add-on dependencies can be defined using the _Depends-On_ metadata field.
+Multiple dependencies may be defined using a comma-separated list of add-on names.
+
+[[example-addon-dependency]]
+.Example: Defining add-on dependency in add-on definition file
+
+----
+# Name: example
+# Description: Shows the use of Depends-On tag
+# Depends-On: anyuid, admin-user
+
+# This add-on depends on anyuid and admin-user add-on
+
+echo Depends on anyuid, admin-user add-on, and requires them to be installed
+----
+
```

Comment:
@anjannath Shouldn't we https://github.com/minishift/minishift/pull/2689/files#diff-68958b57e627de2b2b78ab4d47fefd62R29 add it here instead of putting an example and explain in this section. Like we do for minimum openshift version and put it as optional.

Reply:
@praveenkumar This was done as per this comment https://github.com/minishift/minishift/pull/2689#discussion_r209266353

Reply:
@praveenkumar I'd like to move to this style more generally. At the moment, the way we have the documentation structured requires the user to scroll back up to the example `anyuid` add-on definition above.

Instead, I'd like to see a generic example add-on _and_ include discrete examples for each add-on command/metadata field with their documentation. This allows the reader to go directly to the command/field they need to use and see a brief example of its use there.

For the time being, we could use an example in both locations -- I just don't want the reader to be confused if they go directly to the documentation for the feature they want to use, thus missing the example.